apps/system/test/unit/app_window_test.js

```diff
@@ -1960,6 +1960,17 @@ suite('system/AppWindow', function() {
       );
     });
 
+    test('Titilechange event > private browser', function() {
```

Comment:
Funny - I just copy/pasted this from another test suite, that's what I get :)

Corrected in both places.